backend/services/main-service/utils/packed_data/users.json

```diff
@@ -0,0 +1,152 @@
+[
+  {
+    "userId": 1,
+    "username": "user123",
+    "email": "user123@example.com",
+    "password": "hashedpassword1",
```

Comment:
If we're passing this into the database, we need to hash the password. This can be done using:
```
	hashedPassword, err := bcrypt.GenerateFromPassword([]byte("password123/"), bcrypt.DefaultCost)
	if err != nil {
		log.Fatal("Error hashing password:", err)
	}

	user := db.User{
		...
		Password:     string(hashedPassword),
		...
}
```